package/LICENSE

```diff
@@ -0,0 +1,175 @@
+
```

Comment:
I see few new files added under a `package` folder like LICENSE, README. Checking if any reason for it?